README.md

```diff
@@ -1,2 +1,142 @@
-# vpsearch
-Microbiome code for opensourcing
+----------------------------------------------------------------
+vpsearch - Fast Vantage-Point Tree Search for Sequence Databases
+----------------------------------------------------------------
+
+This is a package for indexing and querying a sequence database for fast
+nearest-neighbor search by means of vantage point trees. For reasonably large
+databases, such as [RDP](https://rdp.cme.msu.edu/), this results in sequence
+lookups that are typically 5-10 times faster than other alignment-based lookup
+methods.
+
+## Usage
+
+Given a sequence database (in FASTA format), `vpsearch build` constructs an
+optimized vantage point search tree. Building the tree is a one-time operation
```

Comment:
A citation or Wikipedia reference would be good for the vantage point search tree.